### DISCUSSION TITLE
Add publication under review

### DISCUSSION THREAD

**Discussion by arilwan** [rendercv/rendercv#65] [Q&A] [answered]
Is there a way to add a field to the publication under review (.e.g. [Under Review]..? The `date` filed is mandatory.

**Comment by sinaatalay** [comment, marked as answer]
Yes, you could use the ‘journal’ key for that. It doesn’t have to be the name of a journal; it’s just the name of the field; it could be anything.

Yes, the date field is mandatory. Maybe you could use the submission date for now? I will consider making it an optional field in the next release.
